tools/clang/test/HLSLFileCheck/d3dreflect/bitfield_reflection.hlsl

```diff
@@ -0,0 +1,76 @@
+// RUN: %dxc -E main -T ps_6_0 -HV 2021 -Vd -validator-version 0.0 %s | %D3DReflect %s | FileCheck %s
```

Comment:
I think we might be able to switch to using `-select-validator internal` instead of `-Vd -validator-version 0.0`

Reply:
unknown argument :(